bids-validator/bids_validator/__init__.py

```diff
@@ -4,3 +4,7 @@
 __version__ = get_versions()['version']
 __all__ = ['BIDSValidator']
```

Comment:
Can you manually verify that `bids_validator.__version__` works? If not, do:

```suggestion
__all__ = ['BIDSValidator', '__version__']
```

Reply:
worked for me